doc/conf.py

```diff
@@ -82,6 +82,8 @@
 # a list of builtin themes.
 html_theme = "sphinx_book_theme"
 
+# Avoid indexing by search engines
+html_extra_path = ['../robots.txt']
```

Comment:
This will create a robots.txt file for all documentation that we build (including the one that will be hosted at craylabs.org. I was digging into the documentation a little more (and verified) that readthedocs by default ships a robots.txt: https://docs.readthedocs.io/en/stable/guides/technical-docs-seo-guide.html#seo-robots-txt

This seems to be the case for us, since this file exists: https://smartsim.readthedocs.io/en/latest/robots.txt

Could you dig into this a little more? It might also (for safety) be worthwhile to still figure out a way to make a robots.txt file but have a way to only create the robots.txt file if we know that it's being built via readthedocs

Could you dig into this and

Reply:
for sure!